### PR TITLE
docs: add season of docs 2020 blog post

### PR DESCRIPTION
**Pull request by erickzhao**

### CODE REVIEW

File: data/blog/2020-season-of-docs.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Google Season of Docs
+author:
+- erickzhao
+date: '2020-05-21'
+---
+
+Electron is proud to be participating in the second edition Google Season of Docs initiative, which pairs mentors from open source organizations with technical writers to improve project documentation.
```
Comment (felixrieseberg) [inline data/blog/2020-season-of-docs.md:8]:
```suggestion
Electron is proud to be participating in the second edition of Google's Season of Docs initiative, which pairs mentors from open source organizations with technical writers to improve project documentation.
```

File: data/blog/2020-season-of-docs.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+---
+
+# What is Season of Docs?
+
+![Season of Docs logo](https://developers.google.com/season-of-docs/images/logo/SeasonofDocs_Logo_SecondaryGrey_300ppi.png)
```
Comment (felixrieseberg) [inline data/blog/2020-season-of-docs.md:14]: Optional: Maybe host locally in case that image goes away.

File: data/blog/2020-season-of-docs.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# How do I sign up?
+
+Are you interested in collaborating with us as a technical writer? First, get familiar with Google's [tech writer guide](https://developers.google.com/season-of-docs/docs/tech-writer-guide) for this year's program, and check out the two [project idea drafts](https://github.com/electron/season-of-docs-2020/blob/master/project-ideas.md) that we have prepared.
+
+In order to be selected as Electron's technical writer for Season of Docs, candidates will need to apply on the Google Season of Docs website during the Technical Writer Application phase that is running from June 8 to July 9..
+
+Your application should include a proposal, which is a written document that describes in detail what you plan to achieve on the Electron docs over the course of 3 months. This proposal can either develop on one of the starting points mentioned in our Project Idea doc, or can be something entirely new. Don't know where to start? You can check out last year's [list of accepted proposals](https://developers.google.com/season-of-docs/docs/2019/participants) for inspiration.
+
+
```
Comment (felixrieseberg) [inline data/blog/2020-season-of-docs.md:28]:
```suggestion
```

File: data/blog/2020-season-of-docs.md (hunk 2 above)
Comment (erickzhao) [reply data/blog/2020-season-of-docs.md]: Good idea. Seems like past blog posts have used the GitHub CDN, so I've uploaded the image there instead.

**Pull request merged**